packages/notebook/style/module.css

```diff
@@ -0,0 +1,11 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+/* This CSS file imports dependency CSS files. */
+
+@import '~@jupyterlab/cells/style/index.css';
```

Comment:
Right now, this is the only way these things get on the page. Thus if someone builds jupyterlab with a console, but no notebook, they will have to include the CSS for these things separately. Should the console extension also include them? Why wouldn't the notebook extension include them in it's index.ts as the other extensions have done?